src/interface/widgets/widget.cpp

```diff
@@ -337,68 +337,42 @@ void Visualizer::drawRobot(QPainter &painter, Robot robot, bool ourTeam) {
         painter.drawText(robotpos.x, ypos += 20, QString::fromStdString(getRoleNameForRobot(robot)));
     }
 
+    // Todo : Get working stuff in RobotView
     if (showRobotInvalids && ourTeam) {
         painter.setPen(Qt::red);
         std::string text;
-        if (!robot.hasWorkingGeneva()) {
-            text += "GV ";
-        }
-        if (!robot.hasWorkingDribbler()) {
+        if (!robot->isWorkingDribbler()) {
             text += "DR ";
         }
-        if (!robot.hasWorkingBallSensor()) {
+        if (!robot->isWorkingBallSensor()) {
             text += "BS ";
         }
-        if (robot.isBatteryLow()) {
+        if (robot->isBatteryLow()) {
             text += "BATTERY LOW";
         }
         painter.drawText(robotpos.x, ypos += 20, QString::fromStdString(text));
     }
 
-    if (ourTeam) {
-        if (Constants::FEEDBACK_ENABLED()) {
-            if (robot.hasRecentFeedback()) {
-                // green to indicate feedback is okay
-                painter.setPen(Qt::green);
-                painter.setBrush(Qt::green);
-            } else {
-                // yellow to indicate feedback is not okay
-                painter.setPen(Qt::red);
-                painter.setBrush(Qt::red);
-            }
-            painter.drawEllipse({(int)robotpos.x + 10, (int)robotpos.y - 10}, 2, 2);
-        }
-    }
+    // Todo : Get working feedback in RobotView
```

Comment:
Way to do this is by making feedback processing a roboteam_world thing. Make this an issue (not critical for now).

Reply:
Issue made